tests/tests.rs

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) 2024, Janoš Guljaš <janos@resenje.org>
+// All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+use casbab;
+
+struct Case<'a> {
```

Comment:
Ja licno bih koristio struct sa owned stringovima (ne &str nego bas String), ali bi onda imao gomilu onih "string".to_string() ili "string".into() poziva u inicijalizaciji.

Moje razmilsjanje jne to, jer me smarjau anotirani lifetimeovi, plus mi je nkekao logicno da stuktura ownuje podatke, an e pozajmljuje. Pa ako nekome trebaju, da se ili moveuju ili kloniraju.

Reply:
Jeste, to je tacnije.